clients/vault/tests/vault_integration_tests.rs

```diff
@@ -212,7 +210,6 @@ async fn test_redeem_succeeds() {
 		get_exchange_rate(&vault_provider, DEFAULT_TESTING_CURRENCY).await;
 		get_exchange_rate(&vault_provider, DEFAULT_WRAPPED_CURRENCY).await;
 		get_exchange_rate(&vault_provider, DEFAULT_NATIVE_CURRENCY).await;
-		assert_eq!(keys.len(), 3);
```

Comment:
Is it necessary to completely remove this line? I assume there is now some kind of mismatch and the assertion does not hold anymore. But if that's the case, can't we simply change the number we expect?

Reply:
Yes. Right. Done!

Reply:
Does this still work? Removing lines 211-215 does not have an impact on the outcome of the `test_redeem_succeeds()` integration test? 😳

Reply:
@cr4pt0 did you check if this particular test still works after the changes?

Reply:
Yes. It works. i added that line trying solve the issue that i have with integration tests.
But lines that we removed are not necessary,` test_with_vault` function feed values for tests.
@ebma